src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -90,7 +96,7 @@ public function translate_post_id( $value, $object_id, $meta_key, $single ) {
 
 		$cache[ $cache_key ] = $value;
 
-		return $value;
+		return $single ? $value[0] : $value;
```

Comment:
We probably want to store the amended value (`$value[0]`) in cache - as we directly return it above (line 52) which could lead to unexpected issues if it's an array.

Reply:
Fixed.=